packages/rn-tester/Podfile

```diff
@@ -6,7 +6,7 @@ platform :ios, min_ios_version_supported
 prepare_react_native_project!
 
 USE_FRAMEWORKS = ENV['USE_FRAMEWORKS'] == '1'
-IN_CI = ENV['CI'] == 'true'
```

Comment:
I'm not sure we should change this, tbh. This is the Podfile of the app and it controls Flipper. We don't want flipper in any CI not just the react native one (nobody has to debug the app in CI and build Flipper takes something like 10 min, so it's safe to remove it when building RNTester in all the CIs).